lntest/itest/lnd_onchain_test.go

```diff
@@ -341,6 +341,40 @@ func testAnchorReservedValue(net *lntest.NetworkHarness, t *harnessTest) {
 	// The reserved value is now back in Alice's wallet.
 	aliceBalance := waitForConfirmedBalance()
 
+	// The reserved value should be equal to the required reserve for anchor
+	// channels.
+	walletBalanceResp, err := alice.WalletBalance(
+		ctxb, &lnrpc.WalletBalanceRequest{},
+	)
+	require.NoError(t.t, err)
+	require.Equal(
+		t.t, walletBalanceResp.ReservedBalanceAnchorChan, aliceBalance,
```

Comment:
nit: I think in `Equal` the first parameter (after `t`) is always the expected value and the second parameter is the value to test. So I think this should be swapped for an eventual error message to be correct?
Same with the `Equal()` below.